v5_order_service.go

```diff
@@ -12,8 +12,9 @@ type V5OrderServiceI interface {
 	CreateOrder(V5CreateOrderParam) (*V5CreateOrderResponse, error)
 	AmendOrder(V5AmendOrderParam) (*V5AmendOrderResponse, error)
 	CancelOrder(V5CancelOrderParam) (*V5CancelOrderResponse, error)
-	GetOpenOrders(V5GetOpenOrdersParam) (*V5GetOpenOrdersResponse, error)
+	GetOpenOrders(V5GetOpenOrdersParam) (*V5GetOrdersResponse, error)
 	CancelAllOrders(V5CancelAllOrdersParam) (*V5CancelAllOrdersResponse, error)
+	GetHistoryOrders(V5GetHistoryOrdersParam) (*V5GetOrdersResponse, error)
```

Comment:
To unify with others

```suggestion
	GetHistoryOrders(V5GetHistoryOrdersParam) (*V5GetHistoryOrdersResponse, error)
```

Reply:
So far the response structure seems to be the same, but since the endpoints are different, it would be better to have separate types